tests/TestInputSocket.py

```diff
@@ -179,14 +179,14 @@ def test_input_socket(self):
         self.sanityCheck(tn)
 
         # Check the aircraft name and its version
-        msg = string.split(tn.sendCommand("info"), '\n')
-        self.assertEqual(string.strip(string.split(msg[2], ':')[1]),
-                         string.strip(root.attrib['name']))
-        self.assertEqual(string.strip(string.split(msg[1], ':')[1]),
-                         string.strip(root.attrib['version']))
+        msg = tn.sendCommand("info").split('\n')
+        self.assertEqual(msg[2].split(':')[1],
+                         root.attrib['name'].strip())
+        self.assertEqual(msg[1].split(':')[1],
+                         root.attrib['version'].strip())
```

Comment:
According to my tests, this code has been incorrectly modified. It should rather be
```python
       self.assertEqual(msg[2].split(':')[1].strip(),
                        root.attrib['name'].strip())
       self.assertEqual(msg[1].split(':')[1].strip(),
                        root.attrib['version'].strip())
```
Otherwise the test fails.